serverless.yml

```diff
@@ -3,8 +3,8 @@ service: threebox-address-server
 provider:
   name: aws
   runtime: nodejs10.x
-  stage: develop
-  region: us-west-2
+  stage: ${opt:stage, 'develop'}
```

Comment:
i think before this was just saying default stage was develop, then overwritten by --stage option, although the way you wrote it is probs more explicit

Reply:
this way, we can define it once here and refer to the stage elsewhere as `${self:provider.stage}` instead of recalculating it every time as `${opt:stage, self:provider.stage}`